frontend/src/store/course/actions.ts

```diff
@@ -120,6 +120,38 @@ const getMentorsByCourseId = createAsyncThunk<
   return mentors;
 });
 
+const getMenteesByCourseId = createAsyncThunk<
+  UserDetailsResponseDto[],
+  CourseGetRequestParamsDto,
+  AsyncThunkConfig
+>(
+  ActionType.GET_MENTOR_MENTEES,
+  async (payload, { extra, dispatch, getState }) => {
+    const { coursesApi } = extra;
+    const {
+      course: { isMentor },
+      auth: { user },
+    } = getState();
+    const hasMentoringPermission = (
+      user as UserWithPermissions
+    ).permissions.find(
+      (permission) => permission.key === PermissionKey.MANAGE_MENTORING,
+    );
```

Comment:
Since there is a helper for it
```suggestion
    const hasMentoringPermission = checkHasPermission({
      userPermissions: (user as UserWithPermissions).permissions,
      permissionKeys: [PermissionKey.MANAGE_MENTORING]
    })
```